site/docs/components/dialog/usage.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+Dialogs are interruptive by nature and should be used sparingly. Use Dialog to present or gather critical information that requires immediate action, whether it’s directly or indirectly part of the user’s current task. For example, a warning dialog that appears when a document is closed without being saved—and asks how the user wishes to proceed—is indirectly related to the user’s task.
+
+### Status
+
+For status dialogs you can set the `status` prop to `error`, `warning`, `success`, or `info` to render a status icon in the title and change the border color of the Dialog. For accessibility reasons, you should pass a `role="alertdialog"` to the Dialog component when rendering an alert dialog. You should rely on users' acknowledgment of the message by clicking the action button.
+
+### When to use Dialog
+
+- To notify critical information that requires immediate action.
+
+- Only as the result of a user’s action within their workflow, to ensure it remains relevant and is perceived as high priority.
+
+- To interrupt user's flow.
+
+### When not to use Dialog
```

Comment:
this line is not rendering correctly as header in preview 🤔

Reply:
good spot, there was a rogue space